garden/src/api/gardens/useSearchGardens.ts

```diff
@@ -0,0 +1,75 @@
+import { search } from "@globus/sdk/cjs";
```

Comment:
Are we actually using the Globus SDK for the search logic here?

Reply:
Looks like this was left over from the previous version, I'll remove it in the next commit